src/openrct2/network/Network.cpp

```diff
@@ -31,12 +32,15 @@
 // This string specifies which version of network stream current build uses.
 // It is used for making sure only compatible builds get connected, even within
 // single OpenRCT2 version.
-#define NETWORK_STREAM_VERSION "24"
+#define NETWORK_STREAM_VERSION "25"
 #define NETWORK_STREAM_ID OPENRCT2_VERSION "-" NETWORK_STREAM_VERSION
 
 static Peep* _pickup_peep = nullptr;
 static int32_t _pickup_peep_old_x = LOCATION_NULL;
 
+// General chunk size is 63 kb.
```

Comment:
```suggestion
// General chunk size is 63 KiB.
```
KiB is unambiguously 1024 bytes, unlike kB.